src/integrationTest/java/com/pubnub/api/integration/PublishIntegrationTests.java

```diff
@@ -281,9 +282,9 @@ public void testOrgJsonObject_Post_History() throws PubNubException, JSONExcepti
     @Test
     public void testOrgJsonObject_Get_Receive() throws PubNubException {
         final String channel = random();
-
+        final String expectedUserMessageType = "test";
+        final String expectedSpaceIdValue = "1-to-1_chat";
```

Comment:
Why not create `final SpaceId expectedSpaceId = new SpaceId("")`? I'd operate on the type that we want to operate on :)

Reply:
I like this idea. I will modify it.